src/nob.h

```diff
@@ -1054,10 +1054,25 @@ bool nob_sv_eq(Nob_String_View a, Nob_String_View b)
 // -1 - error while checking if file exists. The error is logged
 int nob_file_exists(const char *file_path)
 {
-#if _WIN32
-    // TODO: distinguish between "does not exists" and other errors
+#ifdef _WIN32    
     DWORD dwAttrib = GetFileAttributesA(file_path);
-    return dwAttrib != INVALID_FILE_ATTRIBUTES;
+    if (dwAttrib == INVALID_FILE_ATTRIBUTES) {
+        if (GetLastError() == 2) return 0;
```

Comment:
I guess `2` is the value for file not existing? Is there any more human-readable constant for it in WinAPI?

Reply:
Yes. Should've checked, especially since it's actually used in `nob_needs_rebuild` :P My bad.